plugin.yaml

```diff
@@ -10,62 +10,318 @@ plugins:
     # URL to archive containing the plugin or name of directory containing
     # the plugin if it is included in the the blueprint directory under the
     # "plugins" directory.
-    source: https://github.com/cloudify-cosmo/cloudify-gcp-plugin/archive/master.zip
+    source: https://github.com/szpotona/cloudify-gcp-plugin/archive/add_disk.zip
 
 node_types:
   cloudify.gcp.nodes.Instance:
     derived_from: cloudify.nodes.Compute
+    properties:
+      use_external_resource:
+        description: >
+          Indicate whether the resource exists and use existing (true) 
+          or if Cloudify should create new resource (false).
+        type: boolean
+        default: false
+      gcp_config:
+        description: >
+          A dictionary of values to pass to authenticate with the GCP API.
+        default: {}
+      image_id:
+        description: >
+          The ID of the image in your GCP account.
+        type: string
+        default: {}
```

Comment:
Is this formatted as URL? I think this is what's explained here (https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/centos-7-vYYYYMMDD). If so, please make the format clear to the user in the description, or link to this page like you did for instace type.